### DISCUSSION TITLE
关于我 - Miao Yu | 于淼

### DISCUSSION THREAD

**Discussion by yufree** [yufree/yufree.cn#298] [General]
关于我 - Miao Yu | 于淼 
 https://yufree.cn/cn/about/

**Comment by yufree** [comment]
`Originally posted by pudding on 2018-02-13 19:00:18
        `
于大侠好，看了你的博客感觉很有意思，但是实在没找到你的邮箱...我就现在这里留言啦，不知道能否把你的部分内容发布到统计之都呢（[https://cosx.org/）？如果方便希望能留\...](https://cosx.org/）？如果方便希望能留个邮箱交流，我的邮箱是 &quot;https://cosx.org/）？如果方便希望能留个邮箱交流，我的邮箱是&quot;)
puddingnnn529@gmail.com ：）

**Comment by yufree** [comment]
`Originally posted by A on 2021-05-31 13:14:23
        `
双非发现了宝藏

**Comment by yufree** [comment]
`Originally posted by A on 2021-05-31 13:15:02
        `
好棒 发现了宝藏

**Comment by yufree** [comment]
`Originally posted by 土木坛子 on 2021-07-21 13:57:52
        `
西奈山伊坎医学院，我以为是一个你的起的名字呢，还真有这个名字......

**Reply by yufree** [reply]
`Originally posted on 2021-07-28 22:06:19
        `
其实是 Carl Icahn
2012年捐了2亿美金换的名字，官方好像给翻译成西奈山爱康医学院了，某种意义上我确实用了个山寨名字。

**Comment by yufree** [comment]
`Originally posted by 青山 on 2022-09-11 05:31:17
        `
最近看《冰雨火》电视剧，里面有一个戒毒所里的医生名叫于淼，挺耳熟的一个名字，再次进入您的博客这才记起来有个博主也叫于淼。

**Comment by LargeSkull** [comment]
喜欢博主准确又得体的文笔，医学生但看得津津有味

**Comment by imjiaoyuan** [comment]
同是环境科学专业的，不过打算转行做生信了，今年考研生信，还是从博主的文章学到了很多东西
